redirects.yaml

```diff
@@ -14,8 +14,7 @@ snap-enterprise-proxy: /snap-store-proxy/en/
 snap-enterprise-proxy/(?P<page>.*): /snap-store-proxy/{page}
 
 # Landsacpe
-landscape/en/landscape-api/?: /landscape/en/api
```

Comment:
This is no 404ing with the new change. This redirect should be kept

Reply:
I've actually added it to the [redirects table](https://discourse.ubuntu.com/t/landscape-documentation-home/23070?_ga=2.192479466.1109813418.1679260968-346780024.1629501030).

But it looks like it's not working, so I'll update the PR